$OpenBSD: patch-setup_py,v 1.3 2017/09/16 06:03:05 akoshibe Exp $
examples/ is apparently used for beta feature modules, so pull in just
the absolutely needed ones.
Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -20,7 +20,14 @@ setup(
     description='Process-based OpenFlow emulator',
     author='Bob Lantz',
     author_email='rlantz@cs.stanford.edu',
-    packages=[ 'mininet', 'mininet.examples' ],
+    packages=[
+        'mininet',
+        'mininet.openbsd'
+    ],
+    py_modules=[
+        'mininet.examples.cluster',
+        'mininet.examples.clustercli'
+    ],
     long_description="""
         Mininet is a network emulator which uses lightweight
         virtualization to create virtual networks for rapid
